cssom-view-1/Overview.bs

```diff
@@ -420,22 +420,27 @@ an associated element or [=pseudo-element=] <var>element</var> and optionally a
 (which is "<code>auto</code>" if omitted),
 the following steps must be run:
 
-<ol>
- <li><a lt="smooth scroll aborted">Abort</a> any ongoing <a>smooth scroll</a> for <var>box</var>.
- <li>If the user agent honors the 'scroll-behavior' property and one of the following are true:
+1. <a lt="smooth scroll aborted">Abort</a> any ongoing <a>smooth scroll</a> for <var>box</var>.
+1. Resolve any pending promise for <var>box</var> with false.
+1. Let <var>scrollPromise</var> be a new promise.
+1. Return <var>scrollPromise</var>, and run the remaining steps <a>in parallel</a>.
+1. If the user agent honors the 'scroll-behavior' property and one of the following is true:
   <ul>
    <li><var>behavior</var> is "<code>auto</code>" and <var>element</var> is not null and its computed value of the
-   'scroll-behavior' property is ''scroll-behavior/smooth''
+   'scroll-behavior' property is ''scroll-behavior/smooth'', or
    <li><var>behavior</var> is <code>smooth</code>
   </ul>
-  ...then perform a <a>smooth scroll</a> of <var>box</var> to <var>position</var>. Once the position has finished updating, emit the <a event>scrollend</a> event.
-  Otherwise, perform an <a>instant scroll</a> of <var>box</var> to <var>position</var>. After an <a>instant scroll</a> emit the <a event>scrollend</a> event.
+  then perform a <a>smooth scroll</a> of <var>box</var> to <var>position</var>;
+  otherwise, perform an <a>instant scroll</a> of <var>box</var> to <var>position</var>.
+1. Wait until the position has finished updating.
+1. If the scroll position changed as a result of this call, emit the <a event>scrollend</a> event.
+1. Resolve <var>scrollPromise</var> with true if it is still in the pending state.
```

Comment:
This seems a bit clumsy to "wait until the position has finished updating" and then check whether another call resulted in the promise no longer being pending. I wonder if we could change the wait step to wait for the position to finish updating or another call to this method to this method to be made.